### PR TITLE
separating edx and caliper logs

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
Now `caliperized logs` will be sent to `/dev/caliper.log` file and edx's default logs will show their default behavior

### CODE REVIEW

File: openedx_caliper_tracking/processor.py [hunk 1]
```diff
@@ -7,14 +8,16 @@
 
 
 logger = logging.getLogger('tracking')
+logger_caliper = logging.getLogger('caliper')
```
Comment (asamolion) [inline openedx_caliper_tracking/processor.py]: Please change the name to `caliper_logger`

File: openedx_caliper_tracking/processor.py [hunk 2]
```diff
@@ -7,14 +8,16 @@
...
 class CaliperProcessor(BaseBackend):
     def __call__(self, event):
         try:
             caliper_event = base_transformer(event)
             related_function = EVENT_MAPPING[event.get('event_type')]
-            return related_function(event, caliper_event)
+            logger_caliper.info(json.dumps(related_function(event, caliper_event)))
+            return event
         except KeyError as ex:
             logger.exception("Missing transformer method implementation for %s" % event.get('event_type'))
```
Comment (asamolion) [inline openedx_caliper_tracking/processor.py]: use `.format()` for string formatting.

**Review by asamolion — CHANGES_REQUESTED** [review]

File: openedx_caliper_tracking/processor.py (hunk 1 above)
Comment (HamzaIbnFarooq) [reply openedx_caliper_tracking/processor.py]: done

File: openedx_caliper_tracking/processor.py (hunk 2 above)
Comment (HamzaIbnFarooq) [reply openedx_caliper_tracking/processor.py]: done

**Comment by asamolion** [comment]
Discuss with @danialmalik and finalize an architecture for v1.0 of the app.

**Review by asamolion — APPROVED** [review]

**Review by danialmalik — APPROVED** [review]

**Pull request merged**